.github/workflows/style.yml

```diff
@@ -0,0 +1,30 @@
+name: Style
+
+on:
+  push:
+    branches: [ ert3 ]
+  pull_request:
+    branches: [ ert3 ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.6', '3.7', '3.8']
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install flake8 pytest
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
```

Comment:
Should we just remove the `if`-statement such that the installation fails if the `requirements.txt` disappears?